python/foglamp/services/core/api/filters.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+import datetime
+import json
+from aiohttp import web
+from foglamp.common.storage_client.payload_builder import PayloadBuilder
+from foglamp.common.configuration_manager import ConfigurationManager
+from foglamp.services.core import server
+from foglamp.services.core import connect
+from foglamp.services.core.scheduler.entities import Schedule, TimedSchedule, IntervalSchedule, ManualSchedule
+from foglamp.common.storage_client.exceptions import StorageServerError
+from foglamp.common import utils
+from foglamp.services.core.api import utils as apiutils
+from foglamp.common import logger
+
+__author__ = "Massimiliano Pinto"
+__copyright__ = "Copyright (c) 2018 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+_help = """
+    -----------------------------------------------------------------------
+    | POST            | /foglamp/filter                               |
+    -----------------------------------------------------------------------
+"""
+
+_LOGGER = logger.setup("filter")
+
+async def add_filter(request):
+    """
+    Create a new filter with a specific plugin
+    
+    :Example:
+     curl -X POST http://localhost:8081/foglamp/filter -d 
+     '{
+        "name": "North_Readings_to_PI_scale_stage_1Filter",
```

Comment:
It is a minor point, but this example gives the wrong impression. A filter can be used in more than one pipeline by more than one service, so please use a name that does not include a service or task in it. E.g. Scale10 for a filter that scales all reading by 10

Reply:
fixed